src/lang-items.md

```diff
@@ -0,0 +1,133 @@
+  - `eh_personality`: `library/panic_unwind/src/emcc.rs` (EMCC)
+  - `eh_personality`: `library/panic_unwind/src/gcc.rs` (GNU)
+  - `eh_personality`: `library/panic_unwind/src/seh.rs` (SEH)
+  - `eh_catch_typeinfo`: `library/panic_unwind/src/emcc.rs` (EMCC)
+  - `panic`: `library/core/src/panicking.rs`
+  - `panic_bounds_check`: `library/core/src/panicking.rs`
+  - `panic_impl`: `library/core/src/panicking.rs`
+  - `panic_impl`: `library/std/src/panicking.rs`
+- Allocations
+  - `owned_box`: `library/alloc/src/boxed.rs`
+  - `exchange_malloc`: `library/alloc/src/heap.rs`
+  - `box_free`: `library/alloc/src/heap.rs`
+- Operands
+  - `not`: `library/core/src/ops/bit.rs`
+  - `bitand`: `library/core/src/ops/bit.rs`
+  - `bitor`: `library/core/src/ops/bit.rs`
+  - `bitxor`: `library/core/src/ops/bit.rs`
+  - `shl`: `library/core/src/ops/bit.rs`
+  - `shr`: `library/core/src/ops/bit.rs`
+  - `bitand_assign`: `library/core/src/ops/bit.rs`
+  - `bitor_assign`: `library/core/src/ops/bit.rs`
+  - `bitxor_assign`: `library/core/src/ops/bit.rs`
+  - `shl_assign`: `library/core/src/ops/bit.rs`
+  - `shr_assign`: `library/core/src/ops/bit.rs`
+  - `deref`: `library/core/src/ops/deref.rs`
+  - `deref_mut`: `library/core/src/ops/deref.rs`
+  - `index`: `library/core/src/ops/index.rs`
+  - `index_mut`: `library/core/src/ops/index.rs`
+  - `add`: `library/core/src/ops/arith.rs`
+  - `sub`: `library/core/src/ops/arith.rs`
+  - `mul`: `library/core/src/ops/arith.rs`
+  - `div`: `library/core/src/ops/arith.rs`
+  - `rem`: `library/core/src/ops/arith.rs`
+  - `neg`: `library/core/src/ops/arith.rs`
+  - `add_assign`: `library/core/src/ops/arith.rs`
+  - `sub_assign`: `library/core/src/ops/arith.rs`
+  - `mul_assign`: `library/core/src/ops/arith.rs`
+  - `div_assign`: `library/core/src/ops/arith.rs`
+  - `rem_assign`: `library/core/src/ops/arith.rs`
+  - `eq`: `library/core/src/cmp.rs`
+  - `ord`: `library/core/src/cmp.rs`
+- Functions
+  - `fn`: `library/core/src/ops/function.rs`
+  - `fn_mut`: `library/core/src/ops/function.rs`
+  - `fn_once`: `library/core/src/ops/function.rs`
+  - `generator_state`: `library/core/src/ops/generator.rs`
+  - `generator`: `library/core/src/ops/generator.rs`
+- Opting out
+  - `unsafe_cell` (relaxes pointer provenance rules, allowing const-to-mut
+    casts): `library/core/src/cell.rs`
+  - `manually_drop` (opts out of implicit destructor call): `library/core/src/mem/manually_drop.rs`
+- Other
+  - `coerce_unsized`: `library/core/src/ops/unsize.rs`
+  - `drop`: `library/core/src/ops/drop.rs`
+  - `drop_in_place`: `library/core/src/ptr.rs`
+  - `clone`: `library/core/src/clone.rs`
+  - `copy`: `library/core/src/marker.rs`
+  - `send`: `library/core/src/marker.rs`
+  - `sized`: `libcrary/ore/msrc/arker.rs`
+  - `unsize`: `library/core/src/marker.rs`
+  - `sync`: `library/core/src/marker.rs`
+  - `phantom_data`: `libcore/marker.rs`
+  - `discriminant_kind`: `library/core/src/marker.rs`
+  - `freeze`: `library/core/src/marker.rs`
```

Comment:
I still think this is not a good fit because it will get outdated so rapidly. Maybe instead you could recommend `rg '#\[lang' library` or looking at https://doc.rust-lang.org/nightly/nightly-rustc/rustc_hir/lang_items/enum.LangItem.html ?

Reply:
Yeah, I was going to suggest that it would be better to have people look at the docs. And suggesting `rg -F '#[lang' library/` is probably a good idea too.